common/djangoapps/student/tests/test_helpers.py

```diff
@@ -13,24 +15,33 @@
 LOGGER_NAME = "student.helpers"
 
 
+@ddt.ddt
 class TestLoginHelper(TestCase):
     """Test login helper methods."""
     def setUp(self):
         super(TestLoginHelper, self).setUp()
         self.request = RequestFactory()
 
-    def test_unsafe_next(self):
+    @ddt.data(
+        "https://www.amazon.com",
+        "favicon.ico",
+        "https://www.test.com/test.jpg",
+        settings.STATIC_URL + "dummy.png"
+    )
+    def test_unsafe_next(self, unsafe_url):
         """ Test unsafe next parameter """
-        unsafe_url = "https://www.amazon.com"
-        with LogCapture(LOGGER_NAME, level=logging.ERROR) as logger:
+        with LogCapture(LOGGER_NAME, level=logging.WARNING) as logger:
             req = self.request.get(reverse("login") + "?next={url}".format(url=unsafe_url))
+            req.META["HTTP_ACCEPT"] = "image/*"  # pylint: disable=no-member
```

Comment:
This always adds the test to the header, meaning it will be rejected regardless of the URL. That's not fully exercising `get_redirect_to()`.